cli/uninstall/uninstall.go

```diff
@@ -82,3 +83,32 @@ func UninstallProgram(program string, binDst string, headerDst string,
 	log.Infof("%s is uninstalled.", program)
 	return err
 }
+
+// GetList generates a list of options to uninstall.
+func GetList(cmdCtx *cmdcontext.CmdCtx) []string {
+	list := []string{}
+	re := regexp.MustCompile("^(?P<prog>(?:tarantool)|(?:tarantool-ee)|(?:tt))_(?P<ver>.*)$")
+
+	cliOpts, err := configure.GetCliOpts(cmdCtx.Cli.ConfigPath)
+	if err != nil {
+		return nil
+	}
+
+	if cliOpts.App.BinDir == "" {
+		return nil
+	}
+
+	installed, err := os.ReadDir(cliOpts.App.BinDir)
```

Comment:
Pls, name this var installedPrograms, so it is easier to understand that we work with programs here and not installed rocks, installed templates, etc.

Reply:
Fixed